cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -200,7 +200,7 @@ action.cyences_notable_event_action.attacker_search = | stats count by UserName
 action.cyences_notable_event_action.attacker_drilldown = `cs_crowdstrike_eventstream` "metadata.eventType"=DetectionSummaryEvent "event.UserName"=$row.UserName$
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = CrowdStrike EventStream
-
+action.cyences_notable_event_action.teams = SOC
```

Comment:
I'll review this teams for all alerts in Excel. Export it with rest endpoint.

Share it from Sharepoint Splunk site. And ask SOC team to review.

Reply:
https://crossrealms.sharepoint.com/:x:/s/Splunkteamsite2/EVTkTr3NLcRFmKaFb8lxK54BtHto6Xj3xWvMmpUWv-W2Ew?e=Mm4olU